lib/watcher/appmetrics.js

```diff
@@ -0,0 +1,37 @@
+exports.worker = function(handle) {
+  var appmetrics = require('appmetrics');
+  var monitoring = appmetrics.monitor();
```

Comment:
I added new commit that disables all probes described in README. However, it's still not clear to me how can I disable the connection to MQTT on localhost.